domain/src/DomainListBuilder.php

```diff
@@ -34,13 +34,10 @@ public function getFormId() {
    */
   public function getOperations(EntityInterface $entity) {
     $operations = parent::getOperations($entity);
-    $destination = drupal_get_destination();
+    $destination = \Drupal::destination()->getAsArray();
     $default = $entity->isDefault();
     $id = $entity->id();
-
-    // Get CSRF token service.
-    $token_generator = \Drupal::csrfToken();
```

Comment:
I've removed it as `$token_generator` is not used anywhere in `getOperations()`. I've check a number of other entity ListBuilders (e.g. `VocabularyListBuilder`) and none use a token.